scripts/speculative_generation_sample.py

```diff
@@ -0,0 +1,77 @@
+from text_generation_server.models import get_model
+from text_generation_server.pb import generate_pb2
+from typing import List
+import time
+import torch
+from transformers import AutoTokenizer
+
+template = "Below is an instruction that describes a task. Write a response that appropriately completes the request.\n\n### Instruction:\n{}\n\n### Response:"
```

Comment:
Generally not sure whether we should include this script as part of TGIS repo

Reply:
let's leave it for now and we can discuss with Nick when we create PR to the main repo